### PR TITLE
regenrate controller code after repository split

### PR DESCRIPTION
**Pull request by kumargauravsharma**
Issue #, if available:

The ElastiCache service controller code needs to be regenerated per the ACK split repositories and updated import paths.
Check compilation and unit tests by running `make test` for `elasticache-controller`.

Description of changes:

* Updated `elasticache-controller/go.mod` file to refer to local `runtime`
* Regenerated ElastiCache controller using 
```
cd $GOPATH/src/github.com/aws-controllers-k8s/code-generator
make build-ack-generate
export SERVICE="elasticache"
make build-controller SERVICE=$SERVICE
```

Testing: `make test` passed.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,8 +2,10 @@ module github.com/aws-controllers-k8s/elasticache-controller
 
 go 1.14
 
+replace github.com/aws-controllers-k8s/runtime => ../runtime
```
Comment (A-Hilaly) [inline go.mod:5]: I don't think we should keep the `replace` statement here, it should be used only for dev purposes. If the `runtime` repository doesn't exist locally, `go build` will complain with `...runtime/go.mod: no such file or directory`. Other than that there is also the risk of using an outdated version/commit of `runtime`.

**Review by A-Hilaly — APPROVED** [review]
LGTM overall 👍. I left one comment

File: go.mod (hunk 1 above)
Comment (jaypipes) [reply go.mod:5]: @A-Hilaly is correct on the above.

Comment (kumargauravsharma) [reply go.mod]: Updated the go.mod file.

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/elasticache-controller
 go 1.14
 
 require (
-	github.com/aws/aws-controllers-k8s v0.0.2
+	github.com/aws-controllers-k8s/runtime v0.0.0-20210204203051-91a6bd53a48c
```
Comment (jaypipes) [inline go.mod:6]: :+1:

**Review by jaypipes — APPROVED** [review]
:+1:

**Pull request merged**